sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -230,6 +240,11 @@ public void validateUserGroupProperties() {
             throw new IllegalArgumentException("One of the User Group Properties must be populated. "
                 + "Please populate azure.activedirectory.user-group.allowed-groups");
         }
+        if (!DEFAULT_GROUP_RELATIONSHIP.equalsIgnoreCase(groupRelationship)
+            && !"transitive".equalsIgnoreCase(groupRelationship)) {
+            throw new IllegalArgumentException("Configuration 'azure.activedirectory.group-relationship' "
```

Comment:
Is it better we call it xxx.user-group.xxx?

Reply:
Okay